guides/common/assembly_using-external-databases.adoc

```diff
@@ -14,8 +14,11 @@ endif::[]
 
 To create and use external databases for {Project}, you must complete the following procedures:
 
+ifndef::foreman-deb[]
 . xref:preparing-a-host-for-external-databases_{context}[].
+// TODO: a {RHEL} vs an {EL}
 Prepare a {RHEL} 8 server to host the external databases.
```

Comment:
```suggestion
Prepare a {EL} 8 server to host the external databases.
```
https://github.com/theforeman/foreman-documentation/blob/f595593663d763a10da7001e6ef29741fb605b57/guides/common/attributes-satellite.adoc#L60

https://github.com/theforeman/foreman-documentation/blob/f595593663d763a10da7001e6ef29741fb605b57/guides/common/attributes-base.adoc#L116

Will this help for your TODO?

Reply:
No, it's exactly why there is a TODO. It would be `an Enterprise Linux` (correct) and `an RHEL` (incorrect).

One solution would be to create a new macro `{an-EL}` or similar that does the right thing, but I'm not sure what's best.

@maximiliankolb any suggestions?

Reply:
suggestion: "Prepare your {EL} host ..."? I know this is a really ugly issue; it also happens with "a orcharhino Proxy"; sometimes even as "{SmartProxy}s -> orcharhino Proxys" instead of "Proxies" (we have an attribute for the latter by now).

Reply:
Any thoughts on side stepping it and saying `a server` instead? Then leave it to the individual section to prepare the server? Looking ahead when we support both EL 8 & 9 at the same time that may (or may not) be easier.

Reply:
"server" is also OK for me. Any thoughts @asteflova ?

Reply:
I'm okay with "server".

(I would actually prefer "server" regardless of the an/a RHEL/EL issue: No need to specify particular details here because those details are included in the procedure you link to.)

Reply:
IMO Server will be more generic. Decrease the readability score. We should play with attributes even if it applies to all builds.

Reply:
I'm with Aneta on reducing the details in the sentence and saying just "Prepare a server...". Another option would be "Prepare a server on {EL} 8 to host...".